components/sup/src/manager/service/mod.rs

```diff
@@ -367,6 +367,9 @@ impl Service {
         spec.binds = self.binds.clone();
         spec.start_style = self.start_style;
         spec.config_from = self.config_from.clone();
+        if let Some(ref password) = self.svc_encrypted_password {
+            spec.svc_encrypted_password = Some(password.clone())
+        }
```

Comment:
OMG, this pattern strikes again 😦 

I _really_ want to refactor the `Service` to just keep a `ServiceSpec` as a member so we can just return that directly. I think this is at least the 3rd time this has bitten us like this.